src/components/utility/Button/Button.jsx

```diff
@@ -4,25 +4,35 @@ import classNames from "classnames";
 import styles from "./Button.module.css";
 
 const buttonColorStyle = {
-  white: {
-    color: "#fff",
-    borderColor: "#fff",
-    hoverBackgroundColor: "white",
-    hoverColor: "#002529",
+  primary: {
+    white: {
+      color: "#fff",
+      borderColor: "#fff",
```

Comment:
I'm fine with these button spec changes; I want to note that they are also relevant to [my comments on button styles in PR#17](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/17#pullrequestreview-2161609932)